packages/flutter/test/gestures/gesture_tester.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter/gestures.dart';
+
+class TestGestureFlutterBinding extends BindingBase with Gesturer {
+}
```

Comment:
since this block is empty it's probably better for it to be `{ }` rather than with a newline.